scripts/system/html/gridControls.html

```diff
@@ -97,9 +88,9 @@ <h2>Voxel edit settings</h2>
             <div class="tools-help-popup" id="voxel-help-popup" >
                 <p>To edit voxels, Voxel Edit Mode needs to be selected.</p>
                 <p>Desktop mode:</p>
-                <p>Click left mouse button to add voxel. Click middle mouse button to remove voxel. Hold mouse button and move mouse to add/remove voxels in a single plane. Plane is determined by the direction where you are looking when first voxel is added/removed (for example look downwards for horizontal plane.</p>
+                <p>Click the left mouse button to add voxel. Click the middle mouse button to remove voxel. Hold the mouse button and move mouse to add/remove voxels in a single plane. Plane is determined by the direction in which you are looking when first voxel is added/removed (for example look downwards to draw in horizontal plane).</p>
                 <p>VR mode:</p>
-                <p>Pressing trigger adds a voxel. Pressing trigger while holding grip removes it. Hold trigger to add multiple voxels in a single plane. Initial direction of controller ray determies the plane. Hold both grips and move your hands together or apart to change size of edit sphere.</p>
+                <p>Press the trigger to add a voxel. Press the trigger while holding grip to remove voxel. Hold the trigger and move controller to add/remove voxels in a single plane. Plane is determined by the direction in which the controller ray points when first voxel is added/removed (for example point downwards to draw in horizontal plane). Hold both grips and move your hands together or apart to change size of edit sphere.</p>
```

Comment:
```suggestion
                <p>Press the trigger to add a voxel. Press the trigger while holding the grip to remove a voxel. Hold the trigger and move the controller to add or remove voxels in a single plane. The plane is determined by the direction in which the controller's ray points when the first voxel is added or removed (for example point downwards to draw in the horizontal plane). Hold both grips and move your hands together or apart to change the size of the edit sphere.</p>
```

Also, "edit sphere"? Is that a thing, or you just mean it resizes the voxel?

Reply:
`remove voxels in a single plane.` should be `remove voxels on a single plane.`; I would argue that you cannot be inside of a 2 dimensional object.
Pretty sure it would either be `controllers` or `controllers'`, `controller's` suggests that the `s` is short for something like `is`.

Reply:
I think it should be "controller's ray" -- a ray that belongs to the controller.  "controllers ray" doesn't make grammatical sense, and "controllers' ray" would imply there are two controllers with a single ray.

Reply:
"Edit sphere" is a spherical brush for editing more than single voxel at once, with dimensions in world coordinate system. It's used when you switch from single voxels to spheres. You can see how its size affects editing in this video:
https://www.youtube.com/watch?v=FZOzcZsGbZM
As for controller's, you are right that in some cases it's a short for "is", like for example in "it's", but here it's a possessive 's:
https://en.wikipedia.org/wiki/English_possessive

Reply:
Yeah makes sense to me. It is not `controllers'` because it refers to only one controller.